pandas/core/frame.py

```diff
@@ -6686,6 +6686,12 @@ def corr(self, method='pearson', min_periods=1):
             to have a valid result. Currently only available for pearson
             and spearman correlation
 
+        tri : {'upper', 'lower'} or None, default : None
+            Whether or not to return the upper / lower triangular
+            correlation matrix
```

Comment:
Could you add an explanation for why a user might want to do this? And an example below using upper or lower?

Reply:
Sure, you mean within the docstring itself right?